src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -355,6 +386,8 @@ final class LazyList[+A] private(private[this] var lazyState: () => LazyList.Sta
   /** LazyList specialization of reduceLeft which allows GC to collect
     *  along the way.
     *
+    * $preservesLaziness
```

Comment:
This method does force the lazy list

Reply:
oops, I think I meant to put this on `scanLeft`